### PR TITLE
DHIS2-11044_ErrorWidget

### PR DESCRIPTION
**Pull request by eirikhaugstulen**
Added documentation of the error widget in the enrollment overview.

### CODE REVIEW

**Comment by karolinelien** [comment]
LGTM 👍

**Review by karolinelien — APPROVED** [review]

**Review by simonadomnisoru — APPROVED** [review]

File: src/user/using-the-capture-app.md [hunk 1]
```diff
@@ -878,6 +878,15 @@ You can reach the enrollment event form by typing in the address bar of your bro
 
 ![](resources/images/capture_app/enrollment-event-view-edit-url.png)
 
+### Error Widget
+
+On the enrollment dashboard, you can view the Error widget.
+
+![](resources/images/capture_app/enrollment-dash-error-widget-1.png)
+
+The error widget will display errors recieved from configured program rules which are not associated with a specific data element or attribute.
```
Comment (j-nguyenn) [inline src/user/using-the-capture-app.md]: NIT: typo here: `received`

File: src/user/using-the-capture-app.md [hunk 2]
```diff
@@ -878,6 +878,15 @@ You can reach the enrollment event form by typing in the address bar of your bro
 
 ![](resources/images/capture_app/enrollment-event-view-edit-url.png)
 
+### Error Widget
+
+On the enrollment dashboard, you can view the Error widget.
```
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest removing this line and building into the sentence below.

File: src/user/using-the-capture-app.md (hunk 1 above)
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest simplifying and removing references to the DHIS2 data model:
`On the enrollment dashboard, the errors widget displays errors related to the current dashboard. The widget shows errors that are not associated with any specific data item.`

File: src/user/using-the-capture-app.md [hunk 3]
```diff
@@ -878,6 +878,15 @@ You can reach the enrollment event form by typing in the address bar of your bro
...
+
+![](resources/images/capture_app/enrollment-dash-error-widget-1.png)
+
+The error widget will display errors recieved from configured program rules which are not associated with a specific data element or attribute.
+If there are no errors, the error widget will not be displayed.
```
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest rewording the _will not be displayed_ to a simpler _is hidden_:
`If there aren't any errors to show for the current dashboard then the widget is hidden.`

**Review by cooper-joe — CHANGES_REQUESTED** [review]

**Comment by eirikhaugstulen** [comment]
Thanks @cooper-joe & @huocha! 
Implemented the changes 👍

**Review by cooper-joe — APPROVED** [review]
